### PR TITLE
Initial draft of post collections feature

### PR DESCRIPTION
**Pull request by nchan0154**
Discussion in #1096

### CODE REVIEW

**Comment by SaptakS** [comment]
I left some comments in the issue as part of the discussion. Apart from that, I would prefer having the collections page center aligned maybe, since there is no right hand side toc, it feels a lot of space on right hand side in 2k+ monitors.

File: src/css/components/_c-hero.scss [hunk 1]
```diff
@@ -50,17 +50,26 @@
 	.template-team &,
 	.template-contribute &,
 	.template-spotlight &,
-	.template-404 & {
+	.template-404 &,
+	.template-collections & {
```
Comment (ericwbailey) [inline src/css/components/_c-hero.scss:54]: Thank you for working with our absolutely wonk layout styling. This is mostly on me 😓

File: src/img/heroes/collections.svg [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<svg width="1318" height="231" viewBox="0 0 1318 231" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ericwbailey) [inline src/img/heroes/collections.svg:1]: Oh wow, thank you for making this!

**Review by ericwbailey — APPROVED** [review]
This is so dang good 😭

**Pull request merged**